web/src/components/product/ProductSelectionProgress.jsx

```diff
@@ -136,26 +137,28 @@ function ProductSelectionProgress() {
   }, [cancellablePromise, setSoftwareProgress, software]);
 
   return (
-    <Center style={{ blockSize: "100vh" }}>
-      <Grid hasGutter>
-        <GridItem sm={8} smOffset={2}>
-          <Card isPlain>
-            <CardBody>
-              <Stack hasGutter>
-                <h1 style={{ textAlign: "center" }}>
-                  {_("Configuring the product, please wait ...")}
-                </h1>
-                <Progress
-                  selectedProduct={selectedProduct}
-                  storageProgress={storageProgress}
-                  softwareProgress={softwareProgress}
-                />
-              </Stack>
-            </CardBody>
-          </Card>
-        </GridItem>
-      </Grid>
-    </Center>
+    <SimpleLayout showOutlet={false} showInstallerOptions={false}>
```

Comment:
Why do you want to hide the installer options here? What if I need to download the logs because something?